packages/flutter_localizations/lib/src/l10n/localizations.dart

```diff
@@ -140,6 +144,8 @@ const Map<String, Map<String, String>> localizations = const <String, Map<String
     'viewLicensesButtonLabel': r'VIEW LICENSES',
     'anteMeridiemAbbreviation': r'AM',
     'postMeridiemAbbreviation': r'PM',
+    'timePickerHourModeAnnouncement': r'Select hours.',
+    'timePickerMinuteModeAnnouncement': r'Select minutes.',
```

Comment:
Why do these two strings end with a full stop? None of the other english translations seem to have it. If they should, should the other languages include it as well?

Reply:
Removed.